### DISCUSSION TITLE
Create own brew-cask for pre-releases.

### DISCUSSION THREAD

**Discussion by mario-fehr** [Sequel-Ace/Sequel-Ace#716] [Ideas] [answered]
It would be possible to create an brew-cask under `homebrew/cask-versions` for current pre-releases. The question is whether this is used and whether it is needed?

**Comment by Kaspik** [comment]
If people are interested in testing beta version regularly, and it wouldn't affect production releases - why not? So instead of `brew install sequel-ace` it would be `brew install sequel-ace beta`?

**Reply by mario-fehr** [reply]
Yes it would be `brew install sequel-ace-beta` or `brew install --cask sequel-ace-beta`

**Comment by mario-fehr** [comment, marked as answer]
```cask "sequel-ace-beta" do
  version "3.1.0,3011:beta1"
  sha256 "9690aa9a267b6c6141ae323a0946ff41f864b6c75a1435853b50b7418599553f"

  url "https://github.com/Sequel-Ace/Sequel-Ace/releases/download/beta/#{version.before_comma}-#{version.after_comma}/Sequel-Ace-#{version.before_comma}-#{version.after_colon}.zip"
  appcast "https://github.com/Sequel-Ace/Sequel-Ace/releases.atom"
  name "Sequel Ace Beta"
  desc "MySQL/MariaDB database management"
  homepage "https://github.com/Sequel-Ace/Sequel-Ace"

  app "Sequel Ace Beta.app"

  zap trash: [
    "~/Library/Application Support/Sequel Ace",
    "~/Library/Caches/com.sequelace.SequelAce",
    "~/Library/Preferences/com.sequelace.SequelAce.plist",
    "~/Library/Saved Application State/com.sequelace.SequelAce.savedState",
  ]
end
```

@Kaspik can u tell my what I need to change at the paths for `zap trash`? And is `app "Sequel Ace Beta.app"` correct?

**Reply by Jason-Morcos** [reply]
Sequel Ace Beta.app is correct
Beta should share no paths with the main app, but can't recall the paths off the top of my head (on mobile). Downloading the beta manually from GitHub and opening should create all the paths
